example/test.d.ts

```diff
@@ -0,0 +1,1013 @@
+declare namespace proto.jspb.test {
+  export class Empty extends jspb.Message {
+    constructor(data?: any[] | null);
+    toObject(includeInstance?: boolean): GlobalObject;
```

Comment:
GlobalObject is a clutz specific hack for aliasing that we shouldn't need with es6.
Although I wonder if we need to do anything special if we're importing a message named Object. I guess if we use `import *` that won't be an issue.

Reply:
Replaced with `{ [key: string]: unknown }` in return values. Currently also that for arguments, though as I type this I'm realizing in arguments it probably needs to be `any` for the values.